appveyor.yml

```diff
@@ -31,4 +31,4 @@ test_script:
 - cmd: >-
     cd c:\dev\networkit\build
     
-    ctest -V --build-config %CONFIGURATION%
+    ctest -V --build-config %CONFIGURATION% -E jupyterNotebook*
```

Comment:
This seems to be a remainder of a previous iteration

Reply:
Correctly spotted.